README.md

```diff
@@ -268,6 +268,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [Ela Conf](http://elaconf.com) *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg" width="30" />
 
     A safe, inclusive tech leadership conference and community for adult (18+) women (cis and trans), trans men, and genderqueer people to connect with and empower one another.
+    
+  - [European Women In Technology](http://www.europeanwomenintech.com)
+  
+    Learn from industry leaders, connect with people from all areas of tech, and maximise your potential with empowering workshops. Let’s change the industry for the better.
```

Comment:
Remove `Let’s change the industry for the better.`

Also this sounds too much like marketing text, we don't represent them, rather we just provide a description of what they do.

Reply:
Alright, will do :)

Reply:
Awesome!